### PR TITLE
Update documentation for running tests

### PR DESCRIPTION
**Pull request by kynan**
Add instructions for installing dependencies and building docs.

### CODE REVIEW

**Comment by ultimateboy** [comment]
Just a heads up that the test failure appears to be the same intermittent failures that #738 caused. Hard to know what's really causing these failures.

File: docs/testing.rst [hunk 1]
```diff
@@ -137,6 +147,31 @@ submitting a pull request, or be prepared to be mail-spammed by CI.
...
+This will generate html documentation in the folder ``~/code/eve.docs/html``,
+which can be overridden with the ``BUILDDIR`` make variable ::
+
+    $ make html BUILDDIR=/path/to/my/docs
+
+Make sure Sphinx_ reports no errors or warnings when running the above.
+
+To preview the documentation, switch to the build directory and run ::
+
+    $ python3 -m http.server
```
Comment (nicolaiarocci) [inline docs/testing.rst]: I usually simply do `$ open <path to index.html>`, which does not need an http server. Might be a good idea to add this option too.

**Comment by kynan** [comment]
@nicolaiarocci Added the alternative you suggested.

**Comment by nicolaiarocci** [comment]
Merged https://github.com/nicolaiarocci/eve/commit/877ef23713cf51cc7b4145f1da2aa16fbcbb712a

**Comment by nicolaiarocci** [comment]
Thanks!

**Pull request closed without merging**